_includes/v21.1/orchestration/start-kubernetes.md

```diff
@@ -1,30 +1,40 @@
-Choose whether you want to orchestrate CockroachDB with Kubernetes using the hosted Google Kubernetes Engine (GKE)  service, the hosted Amazon Elastic Kubernetes Service (EKS), or manually on Google Compute Engine (GCE) or AWS. The instructions below will change slightly depending on your choice.
+Choose whether you want to orchestrate CockroachDB with Kubernetes using the hosted Google Kubernetes Engine (GKE) service, the hosted Amazon Elastic Kubernetes Service (EKS), or manually on Google Compute Engine (GCE) or AWS. The instructions below will change slightly depending on your choice.
 
 - [Hosted GKE](#hosted-gke)
 - [Hosted EKS](#hosted-eks)
 - [Manual GCE](#manual-gce)
 - [Manual AWS](#manual-aws)
 
+{{site.data.alerts.callout_info}}
+The CockroachDB Kubernetes Operator is currently supported for GKE only.
```

Comment:
Should we add openshift here? @keith-mcclellan

Reply:
yes, and we should link to the openshift specific docs. good catch!